ibm/service/power/data_source_ibm_pi_key.go

```diff
@@ -53,25 +57,28 @@ func DataSourceIBMPIKey() *schema.Resource {
 }
 
 func dataSourceIBMPIKeyRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+
+	// session
 	sess, err := meta.(conns.ClientSession).IBMPISession()
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
+	// arguments
 	cloudInstanceID := d.Get(helpers.PICloudInstanceId).(string)
 
+	// get key
 	sshkeyC := instance.NewIBMPIKeyClient(ctx, sess, cloudInstanceID)
 	sshkeydata, err := sshkeyC.Get(d.Get(helpers.PIKeyName).(string))
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
+	// set attributes
 	d.SetId(*sshkeydata.Name)
-	d.Set(PIKeyDate, sshkeydata.CreationDate.String())
-	d.Set(PIKey, sshkeydata.SSHKey)
-	// TODO: deprecated, to remove
-	d.Set("sshkey", sshkeydata.SSHKey)
-	d.Set(helpers.PIKeyName, sshkeydata.Name)
+	d.Set(Attr_KeyCreationDate, sshkeydata.CreationDate.String())
+	d.Set(Attr_Key, sshkeydata.SSHKey)
+	d.Set("sshkey", sshkeydata.SSHKey) // TODO: deprecated, to remove
```

Comment:
Is sshkey supposed to be mentioned in the markdown doc?

Reply:
And the markdown document would let the users know not to use this attribute any more